README.md

```diff
@@ -332,11 +428,24 @@ prf = "prf" selector
 selector = "*" / 1*DIGIT
```

Comment:
I can't review the line above :(

But the line above should say `prf = "prf:" selector`. (It's missing the `:` in the ABNF)